tests/drivers/i2s/i2s_speed/boards/stm32h573i_dk.overlay

```diff
@@ -0,0 +1,61 @@
+/* i2s-node0 is the transmitter/receiver */
+
+#include <zephyr/dt-bindings/memory-attr/memory-attr-arm.h>
+/ {
+	aliases {
+		i2s-node0 = &i2s1;
+		i2s-node1 = &i2s2;
+	};
+};
+
+&i2s1 {
+	pinctrl-0 = <&i2s1_ws_pa4 &i2s1_ck_pa5 &spi1_miso_pa6 &spi1_mosi_pb5>;
+	pinctrl-names = "default";
+	interrupts = <55 3>;
+	dmas = <&gpdma1 0 7 STM32_DMA_PERIPH_TX
+		&gpdma1 1 6 (STM32_DMA_PERIPH_TO_MEMORY | STM32_DMA_PERIPH_NO_INC |STM32_DMA_MEM_INC | STM32_DMA_PERIPH_16BITS |STM32_DMA_MEM_16BITS | STM32_DMA_PRIORITY_HIGH)>;
```

Comment:
I'm surprised this long line isn't detected by CI scripts. Anyway, it will have to be shorten to be <100 chars.

Then:
- `STM32_DMA_PERIPH_RX` could be used instead of `STM32_DMA_PERIPH_TO_MEMORY | STM32_DMA_MEM_INC`
- `STM32_DMA_PERIPH_NO_INC` could be removed since it's basically 0
- What about adding a  new `STM32_DMA_16BITS` as a shorthand for `STM32_DMA_PERIPH_16BITS | STM32_DMA_MEM_16BITS`